checks/testdata/github-workflow-multiple-unpinned-uses.yaml

```diff
@@ -20,7 +20,5 @@ jobs:
     steps:
       - name: Perform CodeQL Analysis
         uses: github/codeql-action/analyze@v1
-        run: echo "unpinned dependency"
       - name: some name
-        run: echo "unpinned dependency"
```

Comment:
Having both `uses` and `run` in a step in not valid for a workflow. I tested this in a workflow file, and the action failed with:
![image](https://user-images.githubusercontent.com/8961650/140807889-311e1798-74ca-4d3b-8cbd-cdd947c11fa7.png)